### DISCUSSION TITLE
What statements concerning i18n are correct?

### DISCUSSION THREAD

**Discussion by aorlando789** [pimcore/pimcore#11151] [Q&A / Support] [answered]
Hi all,

I have problems with a question that I find a bit confused to complete the certification and it is the following:

![image](https://user-images.githubusercontent.com/22205603/148717177-e9c56945-4bb8-4ed5-a9d8-e48c0641235a.png)

**Comment by ckemptner** [comment, marked as answer]
Where did you find these answers? Please have a look at
https://pimcore.com/docs/pimcore/current/Development_Documentation/Multi_Language_i18n/index.html

How many languages can a document have?
https://pimcore.com/docs/pimcore/current/Development_Documentation/Multi_Language_i18n/Localize_your_Documents.html

What are shared translations?
https://pimcore.com/docs/pimcore/current/Development_Documentation/Multi_Language_i18n/Shared_Translations.html

**Reply by aorlando789** [reply]
Thanks!

**Comment by SankarLabs** [comment]
So one document has always has one exact language? It can't contain multiple languages.

**Reply by ckemptner** [reply]
True that!
